WalletWasabi.Fluent.Generators/Generators/AutoNotifyGenerator.cs

```diff
@@ -85,35 +85,43 @@ public override void Execute(FieldDeclarationSyntax[] syntaxNodes)
 
 		if (addNotifyInterface)
 		{
-			source.Append($@"// <auto-generated />
-#nullable enable
-using ReactiveUI;
-
-namespace {namespaceName}
-{{
-    public partial class {classSymbol.ToDisplayString(format)} : {notifySymbol.ToDisplayString()}
-    {{");
+			source.Append(
+				$$"""
+                // <auto-generated />
```

Comment:
This should be tabs right?
And same down below.

Reply:
Pushed fix